src/test/resources/add.s

```diff
@@ -0,0 +1,34 @@
+; add.s - test file for EduMIPS64.
+;
+; Executes an add and tests the results of the addition.
+;
+; (c) 2018 Andrea Spadaccini and the EduMIPS64 team
```

Comment:
If you want, feel free to change the copyright line in each of the files you created with your own name.

The existing files have this copyright because it was hard to attribute copyright when I released the project, since many people in the team had originally worked on the project, but in this case attribution is easy: you did it. :)

Feel free to do it or not, that's your choice. Thanks!

Reply:
Oh thank you! In order to do this would I commit the changes in my fork and then create another pull request?